src/parameter_bridge.cpp

```diff
@@ -296,25 +296,29 @@ int main(int argc, char * argv[])
     service_execution_timeout, multi_threads
     ](const ros::TimerEvent &) -> void
     {
-      std::map<std::string, std::map<std::string, std::string>> active_ros1_services;
+      try {
+        std::map<std::string, std::map<std::string, std::string>> active_ros1_services;
 
-      XmlRpc::XmlRpcValue payload;
-      if (!ros1_bridge::get_ros1_master_system_state(payload)) {
-        return;
-      }
+        XmlRpc::XmlRpcValue payload;
+        if (!ros1_bridge::get_ros1_master_system_state(payload)) {
+          return;
+        }
 
-      ros1_bridge::get_ros1_services(payload, active_ros1_services);
-      {
-        std::lock_guard<std::mutex> lock(g_bridge_mutex);
-        ros1_services = active_ros1_services;
-      }
+        ros1_bridge::get_ros1_services(payload, active_ros1_services);
+        {
+          std::lock_guard<std::mutex> lock(g_bridge_mutex);
+          ros1_services = active_ros1_services;
+        }
 
-      update_services(
-        ros1_node, ros2_node,
-        ros1_services, ros2_services,
-        service_bridges_1_to_2, service_bridges_2_to_1,
-        services_1_to_2_parameter_name, services_2_to_1_parameter_name,
-        service_execution_timeout, multi_threads);
+        update_services(
+          ros1_node, ros2_node,
+          ros1_services, ros2_services,
+          service_bridges_1_to_2, service_bridges_2_to_1,
+          services_1_to_2_parameter_name, services_2_to_1_parameter_name,
+          service_execution_timeout, multi_threads);
+      } catch (std::runtime_error & e) {
+        fprintf(stderr, "runtime_error in ros1_poll: %s\n", e.what());
```

Comment:
maybe this actually catches the issue, but we do not see the output....i.e. investigate `fprintf` is actually visible when used with `source_amr_ros_bridge && ROBOT=afi-l15 start_amr_ros_bridge`